Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -8,9 +8,9 @@ image:
 description: Unique threat intel technology that automatically serves up relevant
   insights in real time.
 configuration:
-- display: Server URL (e.g., https://api.recordedfuture.com/v2/)
-  name: server
-  defaultvalue: https://api.recordedfuture.com/v2/
+- display: Server URL (e.g., https://api.recordedfuture.com/gw/xsoar/)
+  name: server_url
+  defaultvalue: https://api.recordedfuture.com/gw/xsoar/
```

Comment:
@ChanochShayner This is the new url that we set as default for the api. 

Let me know if you've got any further questions around this.